pkg/util/error_list.go

```diff
@@ -27,10 +27,10 @@ import (
 
 // This helper implements the error interface for []error, but prevents
 // accidental conversion of []error to error.
-type errorList []error
+type ErrorList []error
```

Comment:
NAK.

The whole point of the recent transformations here was to simplify - callers just use []error and SliceToError() (which is still poorly named).